cg/constants/report.py

```diff
@@ -149,11 +149,13 @@
     "median_target_coverage",
     "pct_250x",
     "pct_500x",
+    "gc_dropout",
 ]
 
 REQUIRED_SAMPLE_METADATA_BALSAMIC_TO_WGS_FIELDS = _REQUIRED_SAMPLE_METADATA_BALSAMIC_FIELDS + [
     "median_coverage",
     "pct_60x",
+    "pct_pf_reads_improper_pairs",
```

Comment:
Is it possible to replace "pf" for a more meaningful string?

Reply:
I've removed it from the delivery report model, but will keep it as it it for the metrics since it's the actual name of the metric we get from Picard tools